### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by Danieldotcomcoder**
Add Javascript for popup window in project section for Desktop and mobile version

### CODE REVIEW

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
### Hi @Danieldotcomcoder, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -185,27 +187,24 @@ <h4>2015</h4>
                     </div>
                 </div>
                 <div class="project-description">
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
                 </div>
                 <ul class="project-tags">
```
Comment (Danieldotcomcoder) [inline index.html:191]: project

File: index.html [hunk 2]
```diff
@@ -123,28 +125,28 @@ <h4>Back End Dev</h4>
                         <h4>2015</h4>
                     </div>
                 </div>
```
Comment (Danieldotcomcoder) [inline index.html:127]: test

File: index.html [hunk 3]
```diff
@@ -104,16 +104,18 @@ <h2>LET'S CONNECT</h2>
                 </a>
             </li>
         </ul>
+    </section>
```
Comment (Danieldotcomcoder) [inline index.html:107]: test

File: index.html [hunk 4]
```diff
@@ -104,16 +104,18 @@ <h2>LET'S CONNECT</h2>
...
+    <section class="popup-modal">
+
     </section>
     <section class="works" id="portfolio">
         <div class="card-works">
             <div class="snapshot-portfolio">
-                <img src="img/desktop/0.png" alt="project image">
```
Comment (Danieldotcomcoder) [inline index.html:111]: test

**Pull request merged**